internal/cephfs/volume.go

```diff
@@ -209,8 +271,15 @@ func purgeVolume(ctx context.Context, volID volumeID, cr *util.Credentials, volO
 		volOptions.SubvolumeGroup,
 		"-m", volOptions.Monitors,
 		"-c", util.CephConfigPath,
-		"-n", cephEntityClientPrefix+cr.ID,
-		"--keyfile="+cr.KeyFile)
+		"-n", cephEntityClientPrefix + cr.ID,
+		"--keyfile=" + cr.KeyFile,
+	}
+	if force {
+		arg = append(arg, "--force")
+	}
+
+	_, _, err := execCommand(ctx, "ceph", arg...)
```

Comment:
please use `execCommandErr()` like you did almost everywhere else. Or use `util.ExecCommand()` as `execCommand()` is redundant and will be removed with #1259